src/mgr/MgrCommands.h

```diff
@@ -21,22 +21,38 @@ COMMAND("pg dump_pools_json", "show pg pools info in json only",\
 
 COMMAND("pg ls-by-pool "		\
         "name=poolstr,type=CephString " \
-	"name=states,type=CephChoices,strings=active|clean|down|scrubbing|degraded|inconsistent|peering|repair|recovering|backfill_wait|incomplete|stale|remapped|deep_scrub|backfill|backfill_toofull|recovery_wait|undersized|activating|peered,n=N,req=false ", \
+	"name=states,type=CephChoices,strings=active|clean|down|scrubbing|" \
+        "degraded|inconsistent|peering|repair|recovering|backfill_wait|" \
+        "incomplete|stale|remapped|deep_scrub|backfill|backfill_toofull|" \
+        "recovery_wait|undersized|activating|peered|recovery_toofull|" \
+        "snaptrim|snaptrim_wait|snaptrim_error,n=N,req=false ", \
```

Comment:
I was just thinking about this today.  It might be simpler to just make this CephStrings and drop the validation on the client side.  The mon will parse the state and issue an error if it's not a known pg state anyway... easier to maintain that way.